JavaAssignment/src/pallavi/Q_1.java

```diff
@@ -0,0 +1,45 @@
+package pallavi;
+
+// Write a Java program to print the sum (addition), multiply, subtract, divide and remainder of two numbers given by user
+
+public class Q_1 {
+
+	int addition(int num1, int num2) {
+		int sum = num1 + num2;
+		return sum;
+	}
+
+	int multiplication(int num1, int num2) {
+		int mul = num1 * num2;
+		return mul;
+	}
+
+	int substraction(int num1, int num2) {
+		int sub = num1 - num2;
+		return sub;
+	}
+
+	int division(int num1, int num2) {
+		int div = num1 / num2;
+		return div;
+	}
+
+	int remainder(int num1, int num2) {
+		int rem = num1 % num2;
+		return rem;
+	}
+
+	public static void main(String[] args) {
```

Comment:
This code looks perfectly fine, need to add comments before methods, if you feel you can update method name like multiplicationOperation,.